.pre-commit-config.yaml

```diff
@@ -14,11 +13,21 @@ repos:
       - id: check-yaml
       - id: check-json
       - id: check-xml
-  -   repo: local
-      hooks:
-        -   id: spotless-check-and-format
-            name: spotless check and format
-            entry: ./scripts/hooks/spotless-check-and-format.sh
-            language: script
-            files: ".*.java"
-            pass_filenames: false
+  - repo: local
+    hooks:
+      - id: spotless-check-and-format
+        name: spotless check and format
+        entry: ./scripts/hooks/spotless-check-and-format.sh
+        language: script
+        files: ".*.java"
+        pass_filenames: false
+      - id: license-header-check-and-update
+        name: License header check and update
+        entry: ./scripts/hooks/license-check-and-update.sh
+        language: script
+        exclude: ""
+        always_run: true
```

Comment:
not sure if it was intentional but if you want to run this script only once you should add `pass_filenames: false` - otherwise it will be invoked one time for every single file

Reply:
Specifying `exclude` achieves the same thing. You're right that their documentation suggests using `pass_filenames: false` but honestly I think that's less clear - I expected it to run against all files but without passing filenames. I don't mind changing `exclude: ""` to `pass_filenames: false` though - they're both pretty unclear! :joy:

Reply:
Actually - I just saw we have `pass_filenames: false` above, and I planned to merge the two steps (to half the build time by only invoking maven once). I'll merge the two steps and use the existing definition.